src/routes/routes.js

```diff
@@ -279,4 +279,8 @@ export const modals = {
     isPrivate: false,
     forbiddenTokens: [],
   },
+  changeCommission: {
+    isPrivate: false,
```

Comment:
Should probably be `true`?
Setting it to true will also prevent a crash when refreshing the page when the modal is open.